driver-core/src/test/java/com/datastax/driver/core/ClusterDelegateTest.java

```diff
@@ -22,9 +22,9 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class ClusterDelegateTest extends CCMBridge.PerClassSingleNodeCluster {
+public class ClusterDelegateTest extends CCMTestsSupport {
     @Override
-    protected Collection<String> getTableDefinitions() {
+    public Collection<String> createTestFixtures() {
```

Comment:
Since this is the default behavior of createTestFixes, couldn't this method just be removed?